examples/basics/lists/README.md

```diff
@@ -1,16 +1,18 @@
-# Basics:Lists
+# Basics: Lists
 
-This example will show you how you create and use the different q*ListDefs.
+This example will show you how you create and use the different q\*ListDefs. These lists
+are useful when you want to get a list of all generic objects inside a document of
```

Comment:
Not only generic objects, but I guess that is what the lists are mainly used for.